src/client/entity-editor/submission-section/actions.ts

```diff
@@ -117,81 +117,70 @@ function postSubmission(url: string, data: Map<string, any>): Promise<void> {
 function transformFormData(data:Record<string, any>):Record<string, any> {
 	const newData = {};
 	const nextId = 0;
-	// add new publishers
-	_.forEach(data.Publishers, (publisher, pid) => {
```

Comment:
I don't think there's any harm keeping them, also they help in preview in summary section. though we don't use them in post data since they are already been created and we don't modify them after creation.